styles/_editorjs.scss

```diff
@@ -219,4 +219,36 @@
       background-color: transparent;
     }
   }
+  .codex-editor__redactor{
+    .cdx-block.inline-image{
+      & > .inline-image__control-panel{
+        box-shadow:none;
+      }
+      .inline-image__control-panel{
+       
+        border:1px solid lightgrey ;
+        .panel-scroll{
+          &::-webkit-scrollbar {
+            width: 16px;
```

Comment:
<img width="216" alt="image" src="https://user-images.githubusercontent.com/55176773/220651809-19ab7601-d281-4033-a98c-a40d947688ab.png">
it is not styled like <main /> tag scroollbar, add border radius and make it thinner, use rems instead of px